scripts/perftest/datagen/genDimensionReductionData.sh

```diff
@@ -25,9 +25,9 @@ then
   exit 1;
 fi
 
-CMD=$1
-BASE=$2/dimensionreduction
-MAXMEM=$3
+CMD=${1:-systemds}
```

Comment:
i am not sure what this means. can you explain?

Reply:
This sets a default value if the Arg is null